server/src/main/java/org/opensearch/cluster/decommission/DecommissioningFailedException.java

```diff
@@ -52,4 +53,9 @@ public void writeTo(StreamOutput out) throws IOException {
     public DecommissionAttribute decommissionAttribute() {
         return decommissionAttribute;
     }
+
+    @Override
+    public RestStatus status() {
+        return RestStatus.BAD_REQUEST;
```

Comment:
Here https://github.com/opensearch-project/OpenSearch/blob/main/server/src/main/java/org/opensearch/cluster/decommission/DecommissionService.java#L271 it seems its not a bad request due to which the exception is thrown. Should we use a different exception over here?

Reply:
Hmm right. Will fix this in #5093 PR and would rather throw `NodeClosedException`

Reply:
Lets break down to use specific exceptions and use 400/403 as needed in those relevant cases

Reply:
This PR just attempts to change the status code for `DecommissioningFailedException` and here we are making it 400. Once case which Anshu mentioned where we could rather throw `NodeClosedException` will be fixed in a subsequent PR.